### ISSUE TITLE
Bees eat anything 1.16.5

### ISSUE THREAD

**Issue by genfar1980** [TelepathicGrunt/Bumblezone-Fabric#32]
I saw this was fixed for 1.17 but 1.16.5 still has this issue, and I can not just jump versions due to a lot of mods I use are still not 1.17 or are way to buggy to even consider using.

 Basicaly they will eat anything on your hotbar is your offhand is empty and whats in your offhand if its not empty.

**Comment by TelepathicGrunt** [comment]
Im sorry but 1.16.5 Bumblezone is now unmaintained. I dont have the time to maintain multiple mod versions at once and getting 1.16.5 fabric workspace working kept giving me problems after the whole Java 16 stuff and some toolchain stuff jump to java 11 or whatnot. Another modder can make a pr and give me the built jar and I'll merge and upload that. But for now, 1.16.5 bumblezone isnt going to receive another update going forward

**Comment by genfar1980** [comment]
So I got a working jar but there is no sound folder as that did not seem to come with the github 1.16. source,  but from what I could tell on my server honey and sugar water in a bottle works and other items can not be fed to them anymore. But outside of the sound folder not in the jar it seems to work fine on my server. So how would i give you the jar file for this and would I have to put the sound folder from the older version in the new ones assest first?

**Comment by genfar1980** [comment]
[zipped_up_jar.zip](https://github.com/TelepathicGrunt/Bumblezone-Fabric/files/7025011/zipped_up_jar.zip)
 I added the sound folder from  the other 1.16 file  and retested and sound works too  put it in a zip file to upload it here

**Comment by TelepathicGrunt** [comment]
I saw your pr and i had to close it as it did not seem right. a proper pr to fix the bug would just touch the one line of code in beeinteractivity class but that pr looked like it was shoving a ton of 1.17 code into 1.16 which would blow up the game and make it fail in dev. I went ahead and tried setting up a 1.16.5 workspace and do the fix myself. I had to comment out libcd as that dependency for another mod was blowing up gradle lol. Anyway, try v2.4.11 bumblezone from curseforge as that should fix the issue. That should hold you over till other mods port to 1.17 and become stable
